### PR TITLE
Feature get data from api and show to coin detail screen

### PR DESCRIPTION
**Pull request by tamdev3241**

### CODE REVIEW

File: lib/src/app.dart [hunk 1]
```diff
@@ -1,14 +1,17 @@
 import 'package:flutter/material.dart';
+
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:http/http.dart' as http;
```
Comment (thanhphamtrung) [inline lib/src/app.dart:4]: 10k

File: lib/src/widgets/build_price_change_percentage_icon.dart [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                FontAwesomeIcons.levelDownAlt,
+                color: Colors.red,
+                size: iconSize,
+              )
+            : Icon(
+                FontAwesomeIcons.levelUpAlt,
+                color: constants.ColorsConstant.primaryColor,
+                size: iconSize,
+              )
+        : const SizedBox();
```
Comment (thanhphamtrung) [inline lib/src/widgets/build_price_change_percentage_icon.dart:30]: SizedBox().shink

File: lib/src/app.dart (hunk 1 above)
Comment (tamdev3241) [reply lib/src/app.dart]: I fixed it

File: lib/src/widgets/build_price_change_percentage_icon.dart (hunk 2 above)
Comment (tamdev3241) [reply lib/src/widgets/build_price_change_percentage_icon.dart]: I changed SizedBox() to SizedBox.shrink()

**Review by thanhphamtrung — APPROVED** [review]
LGTM

**Review by thanhphamtrung — APPROVED** [review]
LGTM

**Pull request merged**